src/scanoss/winnowing.py

```
@@ -351,7 +352,71 @@ def __strip_snippets(self, file: str, wfp: str) -> str:
+        return hashlib.md5(opposite_contents).hexdigest()
+
+    def __should_generate_opposite_hash(self, contents: bytes) -> bool:
+        """Determine if an opposite line ending hash (fh2) should be generated.
+        
+        Args:
+            contents: File contents as bytes.
+            
+        Returns:
+            True if fh2 hash should be generated, False otherwise.
+        """
+        has_crlf, has_standalone_lf, has_standalone_cr, has_mixed = self.__detect_line_endings(contents)
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)